isis/src/hayabusa2/apps/hyb2onc2isis/Hayabusa2OncInstrument.trn

```diff
@@ -471,4 +471,14 @@ Group = Compression
   Translation    = (*, *)
 End_Group
 
+# Distance from sun to Hayabusa2
+Group = SolarDistance
+  Auto
+  InputKey       = S_DISTHS
```

Comment:
From Lucille's response, I think we need to get S_DISTRS instead of S_DISTHS from the fits file.

Reply:
Thanks, I missed that. I'll change and re PR

Reply:
Fixed and ready again